ent/src/yb/tools/yb-admin_client_ent.cc

```diff
@@ -146,7 +148,9 @@ Status ClusterAdminClient::ListSnapshots(const ListSnapshotsFlags& flags) {
           HybridTimeToString(HybridTime::FromPB(entry.previous_snapshot_hybrid_time())),
           &json_snapshot, &document.GetAllocator());
     } else {
-      cout << SnapshotIdToString(snapshot.id()) << kColumnSep << snapshot.entry().state() << endl;
+      cout << SnapshotIdToString(snapshot.id()) << kColumnSep 
+           << snapshot.entry().state() << kColumnSep 
+           << HybridTimeToString(HybridTime::FromPB(snapshot.entry().snapshot_hybrid_time())) << endl;
```

Comment:
The line is 102 characters long, please make it <= 100.

Reply:
Sorry for that. Forgot to count the spaces in front of the line. Now should be fixed.